flint.ui/src/components/Chart.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <div id="charts">
+    <div id="chart1">
+      <apexchart
+        type="line"
+        height="230"
+        :options="chartOptionsArea"
+        :series="series"
+      />
+    </div>
+    <div id="chart2">
+      <apexchart
+        type="area"
+        height="130"
+        :options="chartOptionsBrush"
+        :series="series"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import Vue from "vue";
+
+import VueApexCharts from "vue-apexcharts";
+Vue.use(VueApexCharts);
+Vue.component("apexchart", VueApexCharts);
+
+export default {
+  name: "BrushCharts",
+  computed: {
+    received_data() {
+      var dataset = this.$store.state.received_data;
+      var pool_1 = [],
+        pool_2 = [],
+        pool_3 = [],
+        simulation_step = [];
+
+      var lines = dataset.split("\n");
+      lines.splice(0, 4);
+      lines.splice(-4);
```

Comment:
Leave a comment to explain what's happening here

Reply:
`dataset` refers to the `response.data` from the API, which is saved in the state as `received_data`. The above code snippet removes the highlighted parts of the response in the screenshots.

![image](https://user-images.githubusercontent.com/13800316/128171109-b15e1931-f1c5-421f-b833-4ea467459d73.png)

![image](https://user-images.githubusercontent.com/13800316/128170967-048c287e-ec1e-40ef-8db6-b11e22a3d372.png)

Reply:
Thanks! I meant in the code 😅
```
# remove header from response
```